usagov/vote-gov-drupal
# VOTE-1089: Integrate registration type state management

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[VOTE-1089](https://bixal-projects.atlassian.net/browse/VOTE-1089)

## Description (optional)
This PR integrates logic to handle different registration states for each registration type. There is custom logic for States/Territories that fall outside of the expected pattern.

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. Pull branch
2. Run lando retune

### Testing steps
1. Create content for a state/territory page for Alaska, American Samoa, Arkansas, North Dakota
2. Visit each page created and confirm the content appears how it does on the current live Vote.gov site.
3. Check that all bullet points in [VOTE-1089 ticket](https://bixal-projects.atlassian.net/browse/VOTE-1089) description are accurate.
